assets/js/main.js

```diff
@@ -1,4 +1,42 @@
 $(document).ready(function() {
+  // Check if `projects.js` was loaded before this file.
+  // `projects` is defined in `projects.js`.
+  if (typeof projects !== 'undefined') {
+    // Loop over each project
+    projects.forEach(function (project) {
+      if (project.hasOwnProperty('tags')) {
+        // Project has tag(s), so add to every list that matches the tag.
+        project.tags.forEach(function (tag) {
+          addProjectToProjectList(tag, project);
+        });
+      } else {
+        // The project doesn't have any tags, so add it to the "other" section.
+        addProjectToProjectList('other', project);
+      }
+    });
+  } else {
+    // `projects.js` needs to be loaded before this file
+    console.error('Please load `projects.js` before `main.js`!');
+  }
+
+  function addProjectToProjectList(tag, project) {
+    var element = '';
```

Comment:
You think we can use a template generator here? Like Handlebars?

Reply:
But I do agree that is simple as create the template with `element` variable. Just thinking here...